dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2774,3 +2774,66 @@ def test_divisions_with_null_partition(tmpdir, engine):
 
     ddf_read = dd.read_parquet(str(tmpdir), engine=engine, index="a")
     assert ddf_read.divisions == (None, None, None)
+
+
+def test_parquet_pyarrow_write_empty_metadata(tmpdir):
+    # https://github.com/dask/dask/issues/6600
+    dd = pytest.importorskip("dask.dataframe")
```

Comment:
I believe dask.dataframe is already imported as `dd` in the header, so you shouldn't need this in either of the new tests.

Reply:
Thanks, this was lingering after move from a different file (where reqd.)